distribution/kernel/carbon.product

```diff
@@ -13,7 +13,7 @@
    </plugins>
 
    <features>
-      <feature id="org.wso2.carbon.core.runtime" version="4.4.11"/>
+      <feature id="org.wso2.carbon.core.runtime" version="4.4.11.SNAPSHOT"/>
```

Comment:
This should be 4.4.11 since it is the release branch and when the maven release is perform, the version in the carbon.product should denote the release version.